chef/roles/ceph-osd.rb

```diff
@@ -1,5 +1,5 @@
 name "ceph-osd"
 description "Ceph Object Storage Device"
 run_list(
-        "recipe[ceph::osd]"
+        "recipe[ceph::role_ceph_osd]"
```

Comment:
Style/FirstParameterIndentation: Indent the first parameter one step more than the start of the previous line.